app/server/crons/deactivateUsers.js

```diff
@@ -0,0 +1,26 @@
+import { CronJob } from 'cron';
+
+new CronJob(
+  '0 0 12 * * *',
+  Meteor.bindEnvironment(function(err, data) {
+    /* Get all users for whom the deactivation date is set */
+    const users = Meteor.users
+      .find({
+        $and: [
+          { deactivateOn: { $lte: new Date() } },
+          { deactivateOn: { $gt: new Date(0) } },
```

Comment:
Why do we need this `$gt` clause? shouldn't the `$lte` be sufficient? We can also require that the `deactivatedOn` is set, with `exists: true`, or something similar.

Reply:
We need to set the deactivated on records for mongo to be able to find that key in the document. So I had set all the deactivateOn fields to timestamp=0

Reply:
Would `{deactivatedOn: {exists: true}}` have the same result?

Reply:
No it wouldn't. I realised setting deactivateOn by default was unnecessary so I removed it.